File: eshop-integration/eAPI/v1/java/eAPI-v1-connector/src/main/java/cz/monetplus/mips/eapi/v1/connector/entity/EchoRes.java
package cz.monetplus.mips.eapi.v1.connector.entity;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class EchoRes extends SignBase {

	private static final long serialVersionUID = -3825192932302805075L;
	
	@XmlElement
	public int resultCode;

	@XmlElement
	public String resultMessage;

	@Override
	public String toSign() {
		StringBuffer sb = new StringBuffer(); 
		add(sb, dttm);
		add(sb, resultCode);
		add(sb, resultMessage);
		deleteLast(sb);
		return sb.toString();
	}

}

File: examples/eAPI v1.8/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/EchoRes.cs
using System.Text;

namespace CsobGatewayClientExample.Communication.DataObjects
{
    public class EchoRes : SignBase
    {
        public int ResultCode { get; set; }
        public string ResultMessage { get; set; }
        
        public override string ToSign()
        {
            var sb = new StringBuilder();
            Add(sb, DateTime);
            Add(sb, ResultCode);
            Add(sb, ResultMessage);
            DeleteLast(sb);
            return sb.ToString();
        }
    }
}
